### PR TITLE
fix(tabs-extended): label wraps 2 lines then truncates with tooltip

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#5963 

### Description

Tabs extended to allow for two lines of text then truncates and adds tooltip if ellipsis.
Screen Shot 2021-05-20 at 12 16 18 PM

### Changelog

**Changed**

- added inner `div` and `p` so styles could be applied properly for text wrapping and tooltip

### CODE REVIEW

**Review by shixiedesign — APPROVED** [review]
💪 !!

File: packages/styles/scss/components/tabs-extended/_tabs-extended.scss [hunk 1]
```diff
@@ -23,6 +24,71 @@
...
       display: none;
+
+      .#{$prefix}--tabs__nav {
+        position: relative;
+        overflow: visible;
+      }
+
+      .#{$prefix}--tabs__nav-item {
+        min-height: 3rem;
+        width: 10rem;
```
Comment (kennylam) [inline packages/styles/scss/components/tabs-extended/_tabs-extended.scss:35]: Should use spacing/layout tokens here.

**Review by kennylam — COMMENTED** [review]
Looks good @ariellalgilmore! Just one small thing.

File: packages/styles/scss/components/tabs-extended/_tabs-extended.scss (hunk 1 above)
Comment (ariellalgilmore) [reply packages/styles/scss/components/tabs-extended/_tabs-extended.scss]: thanks just updated!

**Review by kennylam — APPROVED** [review]
LGTM thanks @ariellalgilmore!

**Pull request merged**